src/main/resources/db/migration/V35__Alter_table_pedidos_produtos_produtores.sql

```diff
@@ -0,0 +1,2 @@
+alter table pedidos_produtos_produtores
+add column valor real;
```

Comment:
Acho que não precisamos de algo tão preciso quanto `real` aqui. `decimal` já é suficiente.

Ler: https://stackoverflow.com/questions/1056323/difference-between-numeric-float-and-decimal-in-sql-server
https://www.postgresql.org/docs/9.1/datatype-numeric.html

Reply:
Ou se vocês quiserem utilizar o `money` type: https://www.postgresql.org/docs/9.1/datatype-money.html